lib/galaxy/visualization/data_providers/genome.py

```diff
@@ -347,13 +346,15 @@ class TabixDataProvider( FilterableMixin, GenomeDataProvider ):
 
     def open_data_file( self ):
         return ctabix.Tabixfile(self.dependencies['bgzip'].file_name,
-                                index_filename=self.converted_dataset.file_name)
+                                index=self.converted_dataset.file_name)
 
     def get_iterator( self, data_file, chrom, start, end, **kwargs ):
-        start, end = int(start), int(end)
+        # chrom must be a string (not unicode), start/end integers.
```

Comment:
can you add that this is due to pysam version xx requiring this? that way in the future we can check whether or not we can use unicode again.